src/main/java/pl/javastart/task/PalindromeService.java

```diff
@@ -4,6 +4,24 @@ public class PalindromeService {
 
     // nie zmieniaj sygnatury tej metody. Jest ona testowana w PalindromeServiceTest
     public boolean isPalindrome(int[] array) {
+        if (array == null) {
+            return false;
+        } else if (array.length == 0) {
+            return false;
+        } else if (numberComparison(array)) {
+            return true;
+        }
         return false;
     }
-}
+
+    private boolean numberComparison(int[] array) {
```

Comment:
Nazwa meotdy to "liczba porównań". W którym miejscu pojawia się to o czym mówi ta nazwa?
W metodzie powinien być czasownik, który powie "co robi ta metoda"